### PR TITLE
Add easing functions and an example scene

### PR DESCRIPTION
**Pull request by Wall-E-76**
Linked with pull request #80. 
Add new easing functions in the animation module of splib, that allow users to make their animations smoother. The example scene shows the particularities of all of those easing functions.

### CODE REVIEW

File: examples/easingScene.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	confignode = rootNode.addChild("Config")
+	confignode.addObject('RequiredPlugin', name="SofaPython3", printLog=False)
+
+	manager = Sofa.splib.animation.AnimationManager(rootNode)
+	spheres = []
+	for i in range(22):
+		sphere = Sphere(rootNode, "sphere"+str(i), [-20,i,0,0,0,0,1],[i/10.0,i*0.7/10.0,0.9])
+		spheres.append(sphere)
+		Sofa.splib.animation.addAnimation(myAnimate, {"target": sphere}, 10.0, mode="pingpong")
+	Sofa.Simulation.init(rootNode)
```
Comment (damienmarchal) [inline examples/easingScene.py]: This line is not needed.

**Comment by damienmarchal** [comment]
This PR looks really cool, waiting #80 to be merged. If you want to make things easy to can fix #80 into this one.

**Comment by Wall-E-76** [comment]
>     * I'm hesitating to have the animation package in Sofa.splib namespace instead of splib or Sofa.animation
> 
>     * I'm quite sure that the constants could be moved to Sofa.constants instead of Sofa.splib.constant

I wasn't quite sure how to organize this. I thought maybe you would want to recreate the splib library structure, but without making it its own plugin, by simply turning it into a package.
If you think it is better to take the constants out of splib, then it is indeed probably a good idea to delete splib altogether, and opt for a Sofa.animation namespace.

**Comment by damienmarchal** [comment]
Closed because it will be merged in https://github.com/SofaDefrost/plugin.SofaPython3/pull/86

**Pull request closed without merging**